meetings.py

```diff
@@ -111,21 +117,33 @@ def get_meeting_uuids(date, key, secret):
         'page_number': 1
     }
 
-    pages = fetch_records("/metrics/meetings", params, wait=60)  # 1 min rate limit
+    meetings = fetch_records("/metrics/meetings", params, 'meetings', wait=60)  # 1 min rate limit
 
-    meeting_uuids = []
+    for meeting in meetings:
 
-    for page in pages:
-        for meeting in page['meetings']:
-            meeting_uuids.append(meeting['uuid'])
+        topic = ""
+        host_id = ""
 
-    return meeting_uuids
+        series_id = meeting['id']
+        if series_id in series_info.keys():
```

Comment:
you don't need to `.keys()` part of this; `if series_id in series_info:` is enough.